colossalai/utils/profiler/_base_profiler.py

```diff
@@ -0,0 +1,28 @@
+from torch.utils.tensorboard import SummaryWriter
+
+
+class BaseProfiler(object):
+
+    def __init__(self):
+        ...

```

Comment:
this `...` will raise error

Reply:
Which error? I tried a simple test, and it was fine.

Reply:
Not an error, but this function should be removed if no initialization is specified.